Framework/API/inc/MantidAPI/Algorithm.h

```diff
@@ -200,6 +200,11 @@ class MANTID_API_DLL Algorithm : public IAlgorithm,
   /// is provided
   const std::string alias() const override { return ""; }
 
+  /// function to return URL for algorithm documentation; A default
+  /// implementation
+  /// is provided. Override if algorithm not part of the Mantid distribution.
```

Comment:
this was just a remark on english, seems the verb is missing in the second sentence